src/celutil/debug.h

```diff
@@ -10,31 +10,41 @@
 #ifndef _DEBUG_H_
 #define _DEBUG_H_
 
-// Define the DPRINTF macro; g++ supports macros with variable
-// length arguments, so we'll use those when we can.
+#ifdef _MSC_VER
+#include <windows.h>
+#endif
+#include <iostream>
+#include <fmt/printf.h>
+
 #ifdef DPRINTF
 #undef DPRINTF // OSX has DPRINTF
 #endif // DPRINTF
 
-#ifdef __GNUC__
-
 #if !defined(_DEBUG) && !defined(DEBUG)
-#define DPRINTF(level, args...)
+#define DPRINTF(level, format, ...)
 #else
-#define DPRINTF(level, args...) DebugPrint(level, args)
-extern void DebugPrint(int level, const char *format, ...);
-#endif
-
+#define DPRINTF(level, format, ...) DebugPrint(level, format __VA_OPT__(,) __VA_ARGS__)
```

Comment:
drop this define and it passes.

Reply:
could you provide an error log with line? this is perfectly valid c++11 syntax so it should not cause any issues in theory.

Reply:
i saw that particular error in travis but now i have other errors :)